examples/application_commands/options.py

```diff
@@ -0,0 +1,29 @@
+import nextcord
+from nextcord import SlashOption
+from nextcord.interactions import Interaction
+
+client = nextcord.Client()
+
+
+@client.slash_command(guild_ids=[...])  # Limits the guildes
+async def my_select_command(
+    interaction: Interaction, help: str = SlashOption(name="option", description="The option you want")
+):
+    if help == "fun":
+        interaction.response.send_message("Fun!")
+    elif help == "music":
+        interaction.response.send_message(
+            "Music.."
+        )
+    elif help == "mod":
+        interaction.response.send_message(
+            "Moderation is now moved to somewhere else..."
+        )
+    else:
+        interaction.response.send_message("Could not find that help option!")
```

Comment:
This would be another good opportunity to show how to use a list of string options.